lib/ecto/repo/transaction.ex

```diff
@@ -16,7 +16,7 @@ defmodule Ecto.Repo.Transaction do
     case Ecto.Multi.__apply__(multi, name, wrap, return) do
       {:ok, values} -> {:ok, values}
       {:error, {key, error_value, values}} -> {:error, key, error_value, values}
-      {:error, error_value} -> {:error, error_value}
+      {:error, operation} -> raise Ecto.LocalReturnsNotSupportedOnMulti, operation
```

Comment:
Perfect, this is exactly it. I don't think we need a custom exception though. I would instead raise a RuntimeError right here:

```elixir
raise "operation #{inspect operation} is manually rolling back, which is not supported Ecto.Multi"
```

Can you please amend? Thanks!

Reply:
Done! :D

Reply:
@ramortegui I think what @josevalim wanted is to drop the custom exception and raise a RuntimeError here :).